mm2src/mm2_main/src/lp_ordermatch/lp_bot.rs

```diff
@@ -93,7 +93,7 @@ impl From<TradingBotStarted> for TradingBotEvent {
 pub struct RunningState {
     trading_bot_cfg: SimpleMakerBotRegistry,
     bot_refresh_rate: f64,
-    price_url: String,
+    price_urls: AsyncMutex<Vec<String>>,
```

Comment:
I believe we can handle this in a mutex-free manner. If I am not mistaken, this field is accessed from a single location, so there is no need to modify the position of its elements here (it can be done on local-scoped mutable variable instead).

Reply:
done